recipes/grpc-stubs/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "grpc-stubs" %}
+{% set version = "1.24.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/grpc-stubs-{{ version }}.tar.gz
+  sha256: d875d5827e0a1ee6003aa5848243ea5df650f89cef8cf8abd917a27493e970e7
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - grpcio
+    - mypy >=0.730
+    - python >=3.6
+    - typing-extensions
+
+test:
+  imports:
+    - grpc-stubs
```

Comment:
These aren't valid Python imports, and I'm pretty sure that's not what's actually installed by this package. Might be good to see what [another package](https://github.com/conda-forge/pyspark-stubs-feedstock/blob/b028bc25f0532be7613a7b35fa0256e27a5aa9d8/recipe/meta.yaml#L26-L29) does to test the installation of type stubs.

Reply:
Fantastic, this is exactly what I needed. I believe this test has revealed an error in the upstream distribution, so I'll convert this to a WIP while we investigate upstream.

Reply:
I've investigated, the upstream lack of `py.typed` definitions is not a blocker, but tracked at https://github.com/shabbyrobe/grpc-stubs/issues/18

Updated with a workaround test.